bindings/matrix-sdk-ffi/src/platform.rs

```diff
@@ -127,13 +127,17 @@ mod ios {
         otlp_endpoint: String,
     ) -> anyhow::Result<()> {
         let otlp_tracer =
-            setup_remote_jaeger(user, password, otlp_endpoint, "element-x-ios".to_owned())?;
+            super::create_otlp_tracer(user, password, otlp_endpoint, "element-x-ios".to_owned())?;
+
+        let otlp_layer = tracing_opentelemetry::layer().with_tracer(otlp_tracer);
 
         tracing_subscriber::registry()
             .with(EnvFilter::new(configuration))
             .with(fmt::layer().with_ansi(false).with_writer(io::stderr))
-            .with(otlp_tracer)
+            .with(otlp_layer)
```

Comment:
Do we want otlp logs to be affected be the env filter as well? Most tools for reading that data probably have their own filtering, no?

Reply:
Since this is for development I was thinking that, yes, we likely want to be able to muck with this using the env filter.

Reply:
Alright.